src/cmap/connection_pool_events.ts

```diff
@@ -187,9 +187,18 @@ export class ConnectionPoolClearedEvent extends ConnectionPoolMonitoringEvent {
   /** @internal */
   serviceId?: ObjectId;
 
+  interruptInUseConnections?: boolean;
+
   /** @internal */
-  constructor(pool: ConnectionPool, serviceId?: ObjectId) {
+  constructor(
+    pool: ConnectionPool,
+    {
+      serviceId,
+      interruptInUseConnections
+    }: { serviceId?: ObjectId; interruptInUseConnections?: boolean } = {}
```

Comment:
This is a new pattern for command event constructors but I think here it's okay - without it, we'd we be forced to add in optional argument handling logic in the constructor.  I'd rather use an object.